### ISSUE TITLE
Ultra specific testing for debug purposes

### ISSUE THREAD

**Issue by Eh2406** [urbansim/orca_test#1]
Can we assert the value for a specific cell of a table?

**Comment by smmaurer** [comment]
I don't think this can be done with the current `orca_test` implementation, but it could be added.

Would you want to run a test like this repeatedly? If it's just for debugging, you could check it pretty succinctly with a direct call to `orca`:

`assert orca.get_table('table').to_frame().loc[rowid, 'colname'] == value`

Could you say more about the use cases you're thinking of, though?

**Comment by Eh2406** [comment]
Fare, It was a thut I had at the end of the User Conference with you, and so I thout I'd start a conversation.

**Comment by waddell** [comment]
The example above is general - that is, anyone using orca could construct a battery of tests of specific cells in a table using index locations -- but that might not be a very common use case for larger tables.   If there are any specific examples it might help think about what functionality might be missing.

**Comment by Eh2406** [comment]
So I have not thout this thrue. The thout came up as we were discussing debugging of a model. We were talking about the experience of looking at results of a long running model and wondering "where did that value come from." and just having come from the excellent orca_test talk, I was wondering if we could use it to get a breakpoint on the new value showing up.
